src/Func.h

```diff
@@ -281,43 +281,43 @@ class FuncRef {
      * for a Func with multiple outputs. */
     EXPORT Stage operator=(const Tuple &);
 
-    /** Define this function as a sum reduction over the given
-     * expression. The expression should refer to some RDom to sum
-     * over. If the function does not already have a pure definition,
-     * this sets it to zero.
+    /** Define a stage that adds the given expression to this Func. If the
+     * expression refers to some RDom, this performs a sum reduction of the
+     * expression over the domain. If the function does not already have a
+     * pure definition, this sets it to zero.
      */
     // @{
     EXPORT Stage operator+=(Expr);
     EXPORT Stage operator+=(const Tuple &);
     EXPORT Stage operator+=(const FuncRef &);
     // @}
 
-    /** Define this function as a sum reduction over the negative of
-     * the given expression. The expression should refer to some RDom
-     * to sum over. If the function does not already have a pure
-     * definition, this sets it to zero.
+    /** Define a stage that adds the negative of the given expression to this
+     * Func. If the expression refers to some RDom, this performs a sum reduction
+     * of the negative of the expression over the domain. If the function does
+     * not already have a pure definition, this sets it to zero.
      */
     // @{
     EXPORT Stage operator-=(Expr);
     EXPORT Stage operator-=(const Tuple &);
     EXPORT Stage operator-=(const FuncRef &);
     // @}
 
-    /** Define this function as a product reduction. The expression
-     * should refer to some RDom to take the product over. If the
-     * function does not already have a pure definition, this sets it
-     * to 1.
+    /** Define a stage that multiplies the given expression to this Func. If the
```

Comment:
Define a stage that multiplies the given expression to this Func. -> Define a stage that multiplies this Func by the given expression.